app/views/pages/about.html.erb

```diff
@@ -92,7 +92,7 @@
 
     <p>
       The platform was created in 2016. The Twitter account,
-      <a href="https://twitter.com/thepracticaldev">@ThePracticalDev</a> was around before then. We are based out of New York City.
+      <a href="https://twitter.com/thepracticaldev">@<%= SiteConfig.social_media_handles["twitter"] %></a> was around before then. We are based out of New York City.
```

Comment:
missed the one in the `href`. Though technically this HTML page is our DEV about page so we can leave this here anyway

Reply:
I amended this just for the sake of conformity but there are a few hardcoded ones still left in where the page is just too DEV specific and it will get overridden by pages.